app/controllers/admin/account_requests_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Admin::AccountRequestsController < AdminController
+  def index
+    @open_account_requests = AccountRequest.where(confirmed_at: nil).order('created_at').reverse
+    @closed_account_requests = AccountRequest.where.not(confirmed_at: nil).order('confirmed_at').reverse
```

Comment:
To address your concern about issuing 2 ActiveRecord queries, you could replace this with a single query and then partition on `confirmed_at` https://ruby-doc.org/core-2.7.3/Enumerable.html#method-i-partition

But it looks like you are sorting them on different columns after, so although you'd be reducing database queries, you'd have to 2 different sorts in memory vs having the database do it for you. I don't think we will have a ton of data in this table, so it probably won't have a huge performance impact which way we do it now. Pick whatever you think is more readable.

If you keep it the way you have it (2 AR queries + sorting in DB), one suggestion I'd make is to use `order(created_at: :desc)` or `order('created_at DESC')` instead of immediately reversing the result
reference: https://apidock.com/rails/ActiveRecord/QueryMethods/order